frontend/src/pages/projects/screens/detail/pipelines/PipelinesSection.tsx

```diff
@@ -38,6 +38,7 @@ const PipelinesSection: React.FC = () => {
         isEmpty={!installed}
         emptyState={<NoPipelineServer variant="secondary" />}
         showDivider={isPipelinesEmpty}
+        data-testid="pipelines-list-page"
```

Comment:
The `PipelinesSection` is not used as a standalone page.
This test id should be more appropriately named `pipelines-section`.